src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -875,6 +875,7 @@ Object AsyncRPCOperation_sendmany::perform_joinsplit(
             {info.vjsout[0], info.vjsout[1]};
     boost::array<size_t, ZC_NUM_JS_INPUTS> inputMap;
     boost::array<size_t, ZC_NUM_JS_OUTPUTS> outputMap;
+    std::function<int(int)> emptyFunc;
```

Comment:
Since we know that calling MappedShuffle with the default gen results in a crash, I would prefer for the purposes of this fix to not call MappedShuffle, even with an identity gen, in case there are some hidden side-effects.